### PR TITLE
Replaces Net::IP by Net::IP::XS in installation

### PR DESCRIPTION
**Pull request by matsduf**
## Purpose

PR #1119 replaced Net::IP by Net::IP::XS in the code. This PR make the change complete by updating the installation instructions.

## How to test this PR

Verify that all OSs can be successfully installed by following the instructions.

### CODE REVIEW

**Review by mattias-p — DISMISSED** [review]
LGTM

File: docs/Installation.md [hunk 1]
```diff
@@ -108,10 +108,16 @@ Using pre-built packages is the preferred method for Debian and Ubuntu.
 2) Install dependencies from binary packages:
 
    ```sh
-   sudo apt install autoconf automake build-essential cpanminus libclass-accessor-perl libclone-perl libdevel-checklib-perl libemail-valid-perl libfile-sharedir-perl libfile-slurp-perl libidn2-dev libintl-perl libio-socket-inet6-perl liblist-moreutils-perl libmodule-find-perl libmodule-install-perl libmodule-install-xsutil-perl libmoose-perl libmoosex-singleton-perl libnet-ip-perl libpod-coverage-perl libreadonly-perl libssl-dev libldns3 libldns-dev libtest-differences-perl libtest-exception-perl libtest-fatal-perl libtest-nowarnings-perl libtest-pod-perl libtext-csv-perl libtool m4
```
Comment (blacksponge) [inline docs/Installation.md:111]: Why not use `libnet-ip-xs-perl` as it is available in Debian 11 (https://packages.debian.org/bullseye/libnet-ip-xs-perl) and Ubuntu 22.04 (https://packages.ubuntu.com/jammy/libnet-ip-xs-perl)?

File: docs/Installation.md [hunk 2]
```diff
@@ -154,7 +160,7 @@ Using pre-built packages is the preferred method for Debian and Ubuntu.
 5) Install dependencies from binary packages:
 
    ```sh
-   pkg install devel/gmake libidn2 p5-App-cpanminus p5-Class-Accessor p5-Clone p5-Devel-CheckLib p5-Email-Valid p5-File-ShareDir p5-File-Slurp p5-IO-Socket-INET6 p5-List-MoreUtils p5-Locale-libintl p5-Module-Find p5-Module-Install p5-Module-Install-XSUtil p5-Moose p5-MooseX-Singleton p5-Net-IP-XS p5-Pod-Coverage p5-Readonly p5-Test-Differences p5-Test-Exception p5-Test-Fatal p5-Test-NoWarnings p5-Test-Pod p5-Text-CSV net-mgmt/p5-Net-IP dns/ldns
```
Comment (blacksponge) [inline docs/Installation.md:157]: I don't see Net::IP::XS explicitly installed for freebsd, is it normal?

**Review by blacksponge — COMMENTED** [review]
Looks good to me, I do have a suggestion and a question.

File: docs/Installation.md [hunk 3]
```diff
@@ -108,7 +108,7 @@ Using pre-built packages is the preferred method for Debian and Ubuntu.
...
 
    ```sh
-   sudo apt install autoconf automake build-essential cpanminus libclass-accessor-perl libclone-perl libdevel-checklib-perl libemail-valid-perl libfile-sharedir-perl libfile-slurp-perl libidn2-dev libintl-perl libio-socket-inet6-perl liblist-moreutils-perl libmodule-find-perl libmodule-install-perl libmodule-install-xsutil-perl libmoose-perl libmoosex-singleton-perl libnet-ip-perl libpod-coverage-perl libreadonly-perl libssl-dev libldns3 libldns-dev libtest-differences-perl libtest-exception-perl libtest-fatal-perl libtest-nowarnings-perl libtest-pod-perl libtext-csv-perl libtool m4
+   sudo apt install autoconf automake build-essential cpanminus libclass-accessor-perl libclone-perl libdevel-checklib-perl libemail-valid-perl libfile-sharedir-perl libfile-slurp-perl libidn2-dev libintl-perl libio-socket-inet6-perl liblist-moreutils-perl libmodule-find-perl libmodule-install-perl libmodule-install-xsutil-perl libmoose-perl libmoosex-singleton-perl libnet-ip-xs-perl libpod-coverage-perl libreadonly-perl libssl-dev libldns3 libldns-dev libtest-differences-perl libtest-exception-perl libtest-fatal-perl libtest-nowarnings-perl libtest-pod-perl libtext-csv-perl libtool m4
```
[the remark this answers is not in the document]
Comment (matsduf) [reply docs/Installation.md:111]: I do not fully understand the question. For Debian and Ubuntu the instructions provide two methods for installation, deb packages and CPAN. The list above is for the CPAN installation, and there it is meaningful.

If you say that the CPAN method should be removed, then that is out of scope of this PR.

[the remark this answers is not in the document]
Comment (mattias-p) [reply docs/Installation.md:111]: Right. I forgot we had two instructions for Debian/Ubuntu.

**Review by mattias-p — APPROVED** [review]

**Review by blacksponge — APPROVED** [review]

**Pull request merged**